### ISSUE TITLE
Diet menu opens instead of vampire skill menu

### ISSUE THREAD

**Issue by SoloBakvet** [TeamLapen/VampirismIntegrations#58]
## Versions
- Minecraft:  1.16.5                  
- Forge:  36.2.20                      
- Vampirism: 1.8.4
- Integrations: 1.6.1
- Diet: 1.0.1.1

## Issue Description

When a vampire player opens his inventory and presses the fang button to open the skill menu,
it still opens the diet menu.

## Reproduce Steps

1. Make a vampire player
2. Open the inventory
3. Press the fang button to open the skill menu

## Additional Information

**Comment by maxanier** [comment]
Thank you for reporting. I will have a look
If you assign a keybinding to the skill menu, you can still access the skill menu, right?

**Comment by SoloBakvet** [comment]
Yes, the skill menu works fine. The part where vampires don't gain nutrition points for eating food works, it's only that the two buttons still seem to overlap and I assumed the integration also removed the diet button for vampire players. If this is not the case, I am sorry for wasting your time.

**Comment by maxanier** [comment]
It does not, but maybe it should. But the problem also affects hunter players.
Maybe it would be good to somehow automatically change the button location if Diet is installed.
In  the meantime you could do this manually by changing `overrideGuiSkillButtonY`  and X in vampirism-client.toml
